UPGRADE-1.12.md

```diff
@@ -1,3 +1,15 @@
+# UPGRADE FROM `v1.12.0` TO `v1.12.1`
```

Comment:
So this PR should be opened to 1.12 branch 🤔 But it is to 1.13 because of this deprecation I guess, so you should move these notes to the proper upgrade file.

Reply:
What is more, probably the fix should be opened to 1.12, but only the deprecation to 1.13 🤔